src/java_bytecode/java_local_variable_table.cpp

```diff
@@ -651,11 +696,21 @@ void java_bytecode_convert_methodt::setup_local_variables(
   method_with_amapt dominator_args(m, amap);
   dominator_analysis(dominator_args);
 
+#ifdef DEBUG
```

Comment:
Macro `DEBUG` is by default not defined, so even if you use `--verbosity 10` you won't see these three messages unless you explicitly define it. Try to compile `cbmc` with `DEBUG` defined and you will see many other conveniently hidden messages, like these.